tree-sitter.json

```diff
@@ -11,6 +11,9 @@
         "pkl"
       ],
       "injection-regex": "^pkl$",
+      "highlights": "queries/highlights.scm",
+      "injections": "queries/injections.scm",
+      "locals": "queries/locals.scm",
```

Comment:
Unrelated: was getting warnings when running `tree-sitter generate`. This fixes it.